components/scream/src/control/atmosphere_driver.cpp

```diff
@@ -14,6 +14,9 @@
 #include "ekat/ekat_parse_yaml_file.hpp"
 #include "ekat/std_meta/ekat_std_utils.hpp"
 
+#include "scream_config.h" // for SCREAM_CIME_BUILD
+#include "control/fvphyshack.hpp"
```

Comment:
I forgot to add this comment...

From the filename I'm guessing this is meant to be a hack that won't be needed in the future?  If so, should we have a comment somewhere making that clear?  If nothing more than to make it easy to track down which code is a hack.

Reply:
I think it's ok. `fvphyshack` is quite verbose anyways, and all the code that (may) need to be revisited is enclosed in `if (fvphyshack)`, which is easy to grep for.

Reply:
Since I'm adding a commit with comment additions, I'll add a comment about fvphyshack.